gamify-impl/src/main/java/ch/heigvd/gamify/api/endpoints/RegistrationApiController.java

```diff
@@ -27,12 +28,14 @@ public RegistrationApiController(RegisteredAppRepository repository) {
     if (exists) {
       return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
     }
+    var token = UUID.randomUUID().toString();
```

Comment:
Not necessarily very elegant, but `UUID.randomUUID()` uses a `SecureRandom` internally and provides quite a lot of entropy.